packages/common/src/plugins/requestValidation/useHTTPValidationError.ts

```diff
@@ -7,8 +7,10 @@ export function useHTTPValidationError(): Plugin {
       return ({ valid, result }) => {
         if (!valid) {
           result.forEach((error) => {
-            error.extensions.http = {
-              status: 400,
+            if (error.extensions) {
```

Comment:
Maybe we can set an empty object to `extensions` to keep the behavior.

Reply:
I've tried that but unfortunately GraphQLError's `extensions` property is readonly and I was unable to find any way to change it.

Reply:
Maybe this works?
```ts
if (!error.extensions) {
 Object.defineProperty(error, 'extensions', {
   value: {}
 })
}
```

Reply:
@nfearnleypro

Reply:
I tried that as well. Unfortunately it doesn't let me use defineProperty because it's set to `changeable: false`.